appveyor.yml

```diff
@@ -21,10 +19,17 @@ platform:
 install:
   - set PATH=%PYTHON%;%PYTHON%\Scripts;%PATH%
   - echo %PATH%
+  - python --version
+  - git submodule update --init
   - pip install -r requirements-dev.txt
-  - fab bootstrap
+  - python setup.py install
 
 test_script:
-  - python --version
-  - fab --version
-  - fab test
+  - pytest tests/test_basic_int.py
+  - pytest tests/test_basic_str.py
+  - pytest tests/test_cells_and_edges.py
+  - pytest tests/test_h3.py
+  - pytest tests/test_memview_int.py
+  - pytest tests/test_numpy_int.py
+  - pytest tests/test_polyfill.py
+  - pytest tests/test_to_multipoly.py
```

Comment:
Can this be `pytest tests/*.py`?

Reply:
Ideally there should be a pytest.ini file and this is just a single `pytest` command. Example from pydeck [here](https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/pytest.ini).

Reply:
IIRC, I was hoping a single `pytest tests/*` would work, but the test discovery seemed to mess up on appveryor for some reason, which is what led me to listing out each file separately.

I'll give it a shot again, otherwise I'll turn it into an issue to fix in the future.

Reply:
Yeah, with `pytest tests/*.py` I get the following error:

<img width="718" alt="Screen Shot 2020-05-09 at 6 35 02 PM" src="https://user-images.githubusercontent.com/984039/81488887-fd55f100-9223-11ea-9bac-a812a00c8f76.png">

Reply:
I'm not sure what appveyor wants, so it gets an issue: https://github.com/uber/h3-py/issues/104